### PR TITLE
Introduce semantic-logger to json log output

### PR DESCRIPTION
**Pull request by unasuke**
## why
logをJSONで吐くためにsemantic-loggerを導入します

## see also
* https://logger.rocketjob.io/rails
* https://github.com/reidmorrison/rails_semantic_logger

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -76,10 +76,11 @@
   # require "syslog/logger"
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new "app-name")
 
+  # https://logger.rocketjob.io/rails
   if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
-    logger.formatter = config.log_formatter
-    config.logger    = ActiveSupport::TaggedLogging.new(logger)
+    SDTOUT.sync = true
```
Comment (okuramasafumi) [inline config/environments/production.rb:81]:
```suggestion
    STDOUT.sync = true
```

**Comment by okuramasafumi** [comment]
https://logger.rocketjob.io/rails
Maybe it's good to have `amazing_print` gem?

**Comment by unasuke** [comment]
wow

スクリーンショット 2023-10-06 19 15 10

**Review by okuramasafumi — APPROVED** [review]
LGTM!

**Pull request merged**